akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -2890,6 +2890,30 @@ trait FlowOps[+Out, +Mat] {
       FlowShape(merge.in(0), merge.out)
     }
 
+  /**
+   * Merge the given [[Source]]s to this [[Flow]], taking elements as they arrive from input streams,
+   * picking randomly when several elements ready.
+   *
+   * '''Emits when''' one of the inputs has an element available
+   *
+   * '''Backpressures when''' downstream backpressures
+   *
+   * '''Completes when''' all upstreams complete (eagerComplete=false) or one upstream completes (eagerComplete=true), default value is `false`
+   *
+   * '''Cancels when''' downstream cancels
+   */
+  def mergeAll[U >: Out](those: Graph[SourceShape[U], _]*)(eagerComplete: Boolean): Repr[U] = those match {
```

Comment:
should this signature be curried or not @viktorklang @johanandren , or it should be ?
```scala
def mergeAll[U >: Out](eagerComplete: Boolean, those: Graph[SourceShape[U], _]*)
```

Reply:
@hepin1989 I defer to @johanandren :)

Reply:
For the record, discussions going on in https://github.com/akka/akka/pull/31299#discussion_r840351828

Reply:
I will reopen them later on